source/agent/audio/audioMixer/AcmEncoder.cpp

```diff
@@ -175,7 +176,17 @@ int32_t AcmEncoder::SendData(FrameType frame_type,
     frame.additionalInfo.audio.channels = getAudioChannels(frame.format);
     frame.payload = const_cast<uint8_t*>(payload_data);
     frame.length = payload_len_bytes;
-    frame.timeStamp = (AudioTime::currentTime()) * m_rtpSampleRate / 1000;
+
+    int64_t computedTimestamp =  AudioTime::currentTime() * m_rtpSampleRate / 1000; 
+    int64_t sizeFor10Ms = m_rtpSampleRate * 10 / 1000 * frame.additionalInfo.audio.channels; //size for 10 ms, such as 48000 * 10/1000 * 2 = 960
```

Comment:
* frame.additionalInfo.audio.channels is not correct.

Reply:
Thx, would you please tell me why it's not correct?